### ISSUE TITLE
Multiple Keys - Chia Wallet Parsing Error: Wallet Not Updating

### ISSUE THREAD

**Issue by j4s0n** [guydavis/machinaris#836]
**Describe the bug**
When parsing the output of `chia wallet show`, the patterns used by the python `expect()` call are no longer matching the output generated by the `chia` CLI utility.

**To Reproduce**

Steps to reproduce the behavior, e.g.:
1. Using an unraid install (may apply to others), view Machinaris index.
2. Let it farm until a payout is received.
3. Notice that the payout is not represented in the index page's wallet graph, or the wallet subpage view.
4. When viewing the worker log, observed that the following error:

```
2023-01-10 13:09:14 [398] [INFO] Failed to load and send public wallet status because Timeout exceeded.
<pexpect.pty_spawn.spawn object at 0x1501d3384fa0>
command: /chia-blockchain/venv/bin/chia
args: ['/chia-blockchain/venv/bin/chia', 'wallet', 'show']
buffer (last 100 chars): b" XXX\r\n2)    YYY\r\nChoose a wallet key [1-2] ('q' to quit, or Enter to use XXX): "
before (last 100 chars): b" XXX\r\n2)    YYY\r\nChoose a wallet key [1-2] ('q' to quit, or Enter to use XXX): "
after: <class 'pexpect.exceptions.TIMEOUT'>
match: None
match_index: None
exitstatus: None
flag_eof: False
pid: 25755
child_fd: 11
closed: False
timeout: 30
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: None
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0.05
delayafterclose: 0.1
delayafterterminate: 0.1
searcher: searcher_re:
    0: re.compile(b'Wallet height:.*\r\n')
    1: re.compile(b'Wallet keys:.*\r\n')
    2: re.compile(b'Choose wallet key:.*\r\n')
    3: re.compile(b'Choose a wallet key:.*\r\n')
    4: re.compile(b'No online backup file found.*\r\n')
    5: re.compile(b'Connection error.*\r\n')
2023-01-10 13:09:14 [398] [INFO] Job "status_wallets (trigger: interval[0:02:00], next run at: 2023-01-10 13:10:21 MST)" executed successfully
```

**Expected behavior**
The output of the `chia wallet show` command should be correctly parsed.

**System setup:**
 - OS: Unraid
 - Docker version number: 20.10.21
 - Machinaris branch: main
 - Machinaris version number: 0.8.6

**Config**
<details><summary></summary>

```
Default Unriad settings, plus a few plotting drives. One wallet, generated by Machinaris during first run.
```

</details>

**Additional context & screenshots**

Machinaris is running chia 1.6.3.dev0. I can't find this tag/branch on Github, but the output of `chia wallet show` should be defined in `chia/commands/cmd_utils.py` line 165 ([link](https://github.com/Chia-Network/chia-blockchain/blob/c7597a1e82630feecdf3c8fccaa82306c850f046/chia/cmds/cmds_util.py#L165)). On the latest version (not 1.6.3.dev0), we see the following:
```
        prompt: str = (
            f"Choose a wallet key [1-{len(fingerprints)}] ('q' to quit, or Enter to use {logged_in_fingerprint}): "
        )
```

To confirm, when I run `chia wallet show` in my machinaris container, I see this:
```
# docker exec -it machinaris chia wallet show

Active Wallet Key (*):
   -Fingerprint:           XXX
   -Sync Status:           Synced

Wallet Keys:
1)  * XXX
2)    YYY
Choose a wallet key [1-2] ('q' to quit, or Enter to use XXX):
```

This matches. However, the regex used by the `expect()` call in machinaris does not match ([link](https://github.com/guydavis/machinaris/blob/95c8309f6e2154f3a0b2af2d5990edd7217a7877/api/commands/chia_cli.py#L94)):

```
        i = child.expect(["Wallet height:.*\r\n", "Wallet keys:.*\r\n", "Choose wallet key:.*\r\n", 
            "Choose a wallet key:.*\r\n", "No online backup file found.*\r\n", "Connection error.*\r\n"], timeout=90)
```

Machinais is looking for 'Choose a wallet key:' not 'Choose a wallet key' (the trailing ':' is the problem).

This started happening on Jan 4th -- I don't see any changes to Machinaris or Chia at that time, so I'm not sure what the root cause was.

**Comment by guydavis** [comment]
Greetings! Thanks for the detailed write up.  All these details help show the issue quite clearly.  Yes, Chia regularly changes their CLI behaviors so I will need to update to correct this case for [multiple keys](https://github.com/guydavis/machinaris/wiki/Keys#multiple-keys).

**Comment by guydavis** [comment]
Available now in the Machinaris [v0.8.7 release](https://github.com/guydavis/machinaris/releases/tag/0.8.7).
